frontend/app/model/Eventbrite.scala

```diff
@@ -201,7 +204,18 @@ object Eventbrite {
       if EBEvent.providerWhitelist.contains(provider)
     } yield provider
 
-    val mainImageUrl: Option[String] = description.flatMap(desc => "<!--\\s*main-image: (.*?)\\s*-->".r.findFirstMatchIn(desc.html).map(_.group(1)) )
+    val mainImageUrl: Option[Uri] = for {
+      desc <- description
+      m <- """\smain-image:\s*(.*?)\s""".r.findFirstMatchIn(desc.html)
+      uri <- Try(Uri.parse(m.group(1))) match {
+        case Success(uri) =>
+          Logger.debug(s"Event $id main-image source url: $uri")
```

Comment:
I don' think we need to log the success(or we need to change the log level in Production!).